src/api/app/views/webui/user/_basic_info.html.haml

```diff
@@ -47,18 +47,24 @@
   %p= render_as_markdown(user.biography)
 
   .mt-4
-    - if User.session
+    - if User.session && user.email.present?
       = mail_to(user.email, title: "Send mail to #{user.name}", class: 'd-block') do
         %i.fas.fa-envelope.me-1
         = user.email
+    - else
```

Comment:
This will be true if you are not logged in...

Reply:
Fixed